portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,15 +26,7 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
   
-  private ArrayList<String> messages;
-  
-  @Override
-  public void init() {
-    messages = new ArrayList<String>();
-    messages.add("Lovely weather we're having.");
-    messages.add("How was your day?");
-    messages.add("Honey, I'm home!");
-  }
+  private ArrayList<String> messages = new ArrayList<String>();
```

Comment:
Nit: `private List<String> messages = new ArrayList<String>();` is preferred (aka program against interfaces not implementations)